.gitignore

```diff
@@ -5,10 +5,14 @@ lib-cov
 *.out
```

Comment:
Opening this PR to add support for different content types in the Bulk module to be able to insert binary data in bulk. Realized it wasnt supported, and I really kind of need it

Reply:
@stomita @xgeek-net @tnoonan-salesforce Hey guys! Any idea when the next release of jsforce might be and whether you would consider this feature? it's pretty important for us. Thanks!

Reply:
@phillipthurmond I'm now on the way of releasing 2.0, which is rewritten ES2017 + Flow, and freezing current 1.8.x for accepting additional features. I cannot say the releasing date of 2.0 as it has several issues I found already. The PR will be reviewed again after the release of 2.0.